test/allowlist_for_publicAPI.json

```diff
@@ -1960,6 +1960,14 @@
     "Serialization",
     "wrapper_singleton"
   ],
+  "torch.utils.checkpoint": [
```

Comment:
I am not sure why this is being triggered in your PR ... but anyway this is not the right thing to do. I'd suggest adding an `__all__` in `torch/utils/checkpoint.py` if needed